src/main/java/tars/Tars.java

```diff
@@ -0,0 +1,99 @@
+package tars;
+
+import java.io.IOException;
+import java.util.Scanner;
+
+/**
+ * This class is Chatbot application which handles Task Management, mainly ToDos, Deadline and Event tasks
+ * Users can add the 3 different type of tasks and carry out other functions like mark, unmark and delete
+ * Users can also view all the tasks added in a LIST
+ * This application will run till "bye" command given
+ *
+ * @author csk
+ * @version 1
+ */
+public class Tars {
+    private TaskList taskList;
+    private Storage storage;
+    private Ui ui;
+    private Parser parserHelp;
```

Comment:
Do you have any other parsers in this file? If not, I think call this ``parser`` is easier to understand ah!